osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr422.json

```diff
@@ -2036,6 +2036,46 @@
           "access": "R"
         }
       ]
+    },
+    {
+      "tag": "CONFIGURATION_OBJECT",
+      "description": "Configuration object",
+      "note": null,
+      "class-id": 1,
+      "version": 0,
+      "obis": "0.1.94.31.3.255",
+      "group": "ELECTRICITY",
+      "meterTypes": ["SP","PP"],
+      "properties": {},
+      "attributes": [
+        {
+          "id": 2,
+          "description": "value",
+          "datatype": "structure",
+          "valuetype": "SET_BY_CLIENT",
+          "access": "RW"
+        }
+      ]
+    },
+    {
+      "tag": "RANDOMISATION_SETTINGS",
+      "description": "PHY Randomisation Object",
+      "note": null,
+      "class-id": 1,
+      "version": 0,
+      "obis": "0.1.94.31.12.255",
+      "group": "ELECTRICITY",
+      "meterTypes": ["SP","PP"],
+      "properties": {},
+      "attributes": [
+        {
+          "id": 2,
+          "description": "value",
+          "datatype": "structure",
+          "valuetype": "SET_BY_CLIENT",
+          "access": "RW"
+        }
+      ]
```

Comment:
This object is not specified in DSMR4.2.2, but is needed. Please add a note explaining that this object should really be in this config.

Reply:
Added this note:
_This object configuration is supported by DSMR4 devices but not presented in de DSMR4 specification. This object configuration is added according to the SMR 5.0.0 specification_